data-model/SDP-data-model-templates/codicological-unit_sdp-template.json

```diff
@@ -132,35 +162,142 @@
-        "id": "URI",
-        "note": [
-          "array of strings"
+        ],
+        "attested_date": [
+          {
+            "type": "CV",
+            "dating_method": "CV",
+            "transcription": [
+              {
+                "syr": "note: field name is an ISO language code"
+              }
+            ],
+            "translation": [
+              {
+                "en": "note: field name is an ISO langcode"
+              }
+            ],
+            "date_human_readable": "Conversion of the date into a Gregorian calendar system, but still a human readable label",
+            "date_normalized": "ISO 8601 date/date range"
+          }
         ]
       }
     ],
-    "associated_place": [
+    "inscription": [
       {
-        "relation": "CV",
-        "id": "URI",
-        "note": [
+        "id": "ARK",
+        "type": "CV",
+        "locus": [
           "array of strings"
+        ],
+        "transcription": [
+          {
+            "syr": "note: field name is an ISO language code"
+          }
+        ],
+        "translation": [
+          {
+            "en": "note: field name is an ISO langcode"
+          }
+        ],
+        "attested_person": [
+          {
+            "relation": "CV",
+            "id": "URI",
+            "attested_name": {
+              "transcription": [
+                {
+                  "syr": "note: field name is an ISO language code"
+                }
+              ],
+              "translation": [
+                {
+                  "en": "note: field name is an ISO langcode"
+                }
+              ]
+            },
+            "note": [
+              "array of strings"
+            ]
+          }
+        ],
+        "attested_place": [
+          {
+            "relation": "CV",
+            "id": "URI",
+            "attested_name": {
+              "transcription": [
+                {
+                  "syr": "note: field name is an ISO language code"
+                }
+              ],
+              "translation": [
+                {
+                  "en": "note: field name is an ISO langcode"
+                }
+              ]
+            },
+            "note": [
+              "array of strings"
+            ]
+          }
+        ],
+        "attested_date": [
+          {
+            "type": "CV",
+            "dating_method": "CV",
+            "transcription": [
+              {
+                "syr": "note: field name is an ISO language code"
+              }
+            ],
+            "translation": [
+              {
+                "en": "note: field name is an ISO langcode"
+              }
+            ],
+            "date_human_readable": "Conversion of the date into a Gregorian calendar system, but still a human readable label",
+            "date_normalized": "ISO 8601 date/date range"
+          }
         ]
       }
     ]
   },
-  "associated_dates": [
+  "associated_person": [
```

Comment:
I reordered these. I'm really not sure if they will be used. When would we have persons or places that are not otherwise attested in the ms?